test/Microsoft.AspNet.Mvc.FunctionalTests/ActionResultTests.cs

```diff
@@ -2,30 +2,27 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.IO;
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading.Tasks;
-using System.Xml.Serialization;
 using ActionResultsWebSite;
 using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.TestHost;
 using Xunit;
 
 namespace Microsoft.AspNet.Mvc.FunctionalTests
 {
     public class ActionResultTests
     {
-        private readonly IServiceProvider _provider = TestHelper.CreateServices("ActionResultsWebSite");
+        private const string SiteName = nameof(ActionResultsWebSite);
         private readonly Action<IApplicationBuilder> _app = new Startup().Configure;
-                
+
         [Fact]
         public async Task BadRequestResult_CanBeReturned()
         {
             // Arrange
-            var server = TestServer.Create(_provider, _app);
+            var server = TestHelper.CreateServer(_app, SiteName);
```

Comment:
One other thing that might clean things up dramatically, is it possible to move these two duplicated lines into a constructor of all the functional tests?  This would save a lot of duplication, not sure if that would work, but worth a shot...
```
          public ActionResultTests() {
             _server = TestHelper.CreateServer(_app, SiteName);
             _client = server.CreateClient();
          }
```